_posts/2024-02-21-replacing-default-admin-credentials.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Enhancing Security in OpenSearch 2.12.0: The End of Default Admin Password"
+authors:
+ - dchanp
+date: 2024-02-21
+categories:
+ - community-updates
+ - technical-posts
+meta_keywords: "OpenSearch Security, Security Demo Configuration"
+meta_description: Learn about the changes to Security plugin's demo configuration setup
+has_math: false
+has_science_table: false
+---
+
+In the realm of data management and search engines, OpenSearch has been a key player, offering robust features and a flexible platform for users worldwide. With the release of OpenSearch 2.12.0, a significant security upgrade is set to transform how administrators initially set up their clusters using the security plugin's demo configuration tool.
+
+**The Vulnerability of Default Passwords**
```

Comment:
Vulnerability is a scary word that in the realm of the Security Plugin sounds like a CVE-ish. Not saying it is wrong, but given the context perhaps I would say something like "The Problem with Default Passwords". Then, in the first paragraph below, I would start by explaining the problem with them: that sometimes demo clusters end up becoming production ones, and that despite that not being a good practice, or having warnings scattered around config files, documentation and console outputs, there are still cases of default passwords being found in the wild. We want to make sure we tackle this in a more definite way: by not having a default password to begin with.